package/thingino-webui/files/var/www/x/config-audio.cgi

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/haserl
+<%in _common.cgi %>
+<%
+plugin="audio"
+page_title="Audio"
```

Comment:
Backchannel Audio would be a better name given that users will be confused why there are no Audio output settings here.

Reply:
Yes, we alreeady have too many pages and grouping controls by functionality could reduce the number. Audio page might include gpio pin for speaker, level and gain of playback on the speaker, settings for audio filters, backchannel, audio files to play on start and/or events, etc.

Reply:
Given the above thread, @jayfan0 are you going to look at all audio settings?

Reply:
I wasn't planning to. Not as part of this PR anyway. I'm not sure if I'll have the time anytime soon

Reply:
I've renamed the setting (not he page) to Audio Backchannel and added a placeholder for Audio Output Settings to avoid confusion

![image](https://github.com/user-attachments/assets/13b7595b-464f-4abb-a5e2-ebcda6d8cb13)